pkg/controller/container-runtime-config/helpers.go

```diff
@@ -205,25 +206,9 @@ func getManagedKeyCtrCfgDeprecated(pool *mcfgv1.MachineConfigPool) string {
 // nolint: dupl
 func getManagedKeyCtrCfg(pool *mcfgv1.MachineConfigPool, client mcfgclientset.Interface, cfg *mcfgv1.ContainerRuntimeConfig) (string, error) {
 	// Get all the ctrcfg CRs
-	ctrcfgListAll, err := client.MachineconfigurationV1().ContainerRuntimeConfigs().List(context.TODO(), metav1.ListOptions{})
+	ctrcfgList, err := getContainerRuntimeConfigsForPool(client, pool)
```

Comment:
The new function removed the below lines of code from `getManagedKeyCtrCfg`, add these lines back can fix the unit test `TestContainerRuntimeConfigUpdate`

https://github.com/openshift/machine-config-operator/blob/f5d2289db0692156343a4f8e48b654583d1c7352/pkg/controller/container-runtime-config/helpers.go#L212-L215

Reply:
ah, good catch!